lib/spack/docs/build_systems/wafpackage.rst

```diff
@@ -0,0 +1,124 @@
+.. _wafpackage:
+
+----------
+WafPackage
+----------
+
+Like SCons, Waf is a general-purpose build system that does not rely
+on Makefiles to build software.
```

Comment:
What are example packages that use `waf`?

Reply:
`py-py2cairo` and `tut` are the only Waf packages currently in Spack.